app/workers/zync_worker.rb

```diff
@@ -13,6 +13,11 @@ class ZyncWorker
 
   sidekiq_options queue: :zync
 
+  # It is used as the delay, in seconds.
+  sidekiq_retry_in do |count, _exception|
+    10 * (count + 1) # (i.e. 10, 20, 30, 40, 50)
```

Comment:
This is in seconds :/
Which means the last one will be retried in:

10 x (1 + 2 + 3 + ... + 25) = 10x(25x26)/2  = 3250 seconds which is less than one hour.

So if there is an issue in Zync side then it has to be solved in this period of time otherwise the job will be lost

Reply:
oh, I see 😞 Ok, I will change it 👍